### PR TITLE
Fixed only the issue about "BOOLEAN" occurring when importing resources.

### PR DESCRIPTION
**Pull request by Lemon-miaow**
Fixed only the issue about `"BOOLEAN"` occurring when importing resources.
It is now possible to load resource without crashing but still has problems.
I'm a noob so I can't fix the `BEGIN_ARRAY` or `BEGIN_OBJECT` problem about gson further.If my code is too bad please do not scold me, I am sorry :(

## Description

Please carefully read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md) before making any pull requests.
And, **Do not make a pull request to merge into stable unless it is a hotfix. Use the development branch instead.**
## Type of changes

- [x] Bug fix
- [ ] New feature 
- [ ] Enhancement
- [ ] Documentation

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] My pull request is unique and no other pull requests have been opened for these changes
- [x] I have read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md)
- [x] I am responsible for any copyright issues with my code if it occurs in the future.

### CODE REVIEW

File: src/main/java/emu/grasscutter/data/common/DynamicFloat.java [hunk 1]
```diff
@@ -42,11 +53,21 @@ public DynamicFloat(float constant) {
         this.constant = constant;
     }
 
+    public String toString(boolean nextBoolean){
+        String key = String.valueOf(nextBoolean);
+        this.ops = List.of(new StackOp(key));
+        return ops.toString();
+    }
```
Comment (KingRainbow44) [inline src/main/java/emu/grasscutter/data/common/DynamicFloat.java:59]: Add a line break between this and the next function.

File: src/main/java/emu/grasscutter/data/common/DynamicFloat.java [hunk 2]
```diff
@@ -28,7 +36,10 @@ public StackOp(String s) {
                 }
             }
         }
-
+        public StackOp(boolean b) {
+            this.op = Op.NEXBOOLEAN;
+            this.bValue = Boolean.parseBoolean(String.valueOf(b));
+        }
```
Comment (KingRainbow44) [inline src/main/java/emu/grasscutter/data/common/DynamicFloat.java:39]: Add a line break between this and the next function.

File: src/main/java/emu/grasscutter/data/common/DynamicFloat.java [hunk 3]
```diff
@@ -1,20 +1,28 @@
...
+import org.jetbrains.annotations.NotNull;
+import org.luaj.vm2.ast.Str;
 
 public class DynamicFloat {
     public static DynamicFloat ZERO = new DynamicFloat(0f);
 
+
     public static class StackOp {
-        enum Op {CONSTANT, KEY, ADD, SUB, MUL, DIV};
+
```
Comment (KingRainbow44) [inline src/main/java/emu/grasscutter/data/common/DynamicFloat.java]: Please revert this whitespace to the original. (delete line 17 & 19)

File: src/main/java/emu/grasscutter/data/common/DynamicFloat.java [hunk 4]
```diff
@@ -42,11 +53,21 @@ public DynamicFloat(float constant) {
         this.constant = constant;
     }
 
+    public String toString(boolean nextBoolean){
```
Comment (KingRainbow44) [inline src/main/java/emu/grasscutter/data/common/DynamicFloat.java]: Add a space between `(` and `{`.

File: src/main/java/emu/grasscutter/utils/JsonUtils.java [hunk 5]
```diff
@@ -44,8 +44,11 @@ public static <T> T decode(JsonElement jsonElement, Class<T> classType) throws J
 
     public static <T> T loadToClass(Reader fileReader, Class<T> classType) throws IOException {
         return gson.fromJson(fileReader, classType);
-    }
 
+
+
+        // return gson.fromJson(fileReader, classType);
+    }
```
Comment (KingRainbow44) [inline src/main/java/emu/grasscutter/utils/JsonUtils.java]: Remove the whitespace + extra redundant lines.

**Review by KingRainbow44 — CHANGES_REQUESTED** [review]

File: src/main/java/emu/grasscutter/utils/JsonUtils.java (hunk 5 above)
Comment (Lemon-miaow) [reply src/main/java/emu/grasscutter/utils/JsonUtils.java]: Thank you for your guidance!

**Comment by ArthurZhou** [comment]
when could this freaking issue got fixed :(

**Review by KingRainbow44 — APPROVED** [review]

**Pull request merged**